BUILD

```diff
@@ -0,0 +1,159 @@
+exports_files(["requirements.txt"])
+
+load("@io_bazel_rules_python//python:python.bzl", "py_library", "py_test")
+load("@pypi_dependencies//:requirements.bzl", "requirement")
+#load("@graknlabs_rules_deployment//pip:rules.bzl", "deploy_pip")
+
+## works for sklearn imported in my_test.py
```

Comment:
are these codes commented out since we haven't made them work yet?

Reply:
Yeah the top comment is, the other I'm removing